excercises/3-synchronization/home.spec.ts

```diff
@@ -1,27 +1,39 @@
-import { browser, element, by } from 'protractor';
+import { browser, element, by , ExpectedConditions} from 'protractor';
+
+const EC = ExpectedConditions;
 
 describe('Protractor Workshop app', function() {
 
 	beforeEach(async function () {
 		await browser.get("/jak-to-zrobic-w-js");
 	});
 
-	xit('should hide GDRP popup after clicking Accept', function(){
-		
+	it('should hide GDRP popup after clicking Accept', async function(){
+		const popupButton = element(by.css('.cookieButton a'));
+		const popupContainer = element(by.css('.cookieConsentContainer'));
+		const expectedStyle = "opacity: -0.02; display: none;";
+		popupButton.click();
```

Comment:
not missing await here?

Reply:
code updated, however I still cannot understand why it only works with doubled button click and not a single click :<